Diablo.dsp

```diff
@@ -44,7 +44,7 @@ RSC=rc.exe
 # PROP Ignore_Export_Lib 0
 # PROP Target_Dir ""
 # ADD BASE CPP /nologo /W3 /GX /O2 /D "WIN32" /D "NDEBUG" /D "_WINDOWS" /YX /FD /c
-# ADD CPP /nologo /Gr /MT /W3 /GX /O1 /D "WIN32" /D "NDEBUG" /D "_WINDOWS" /YX /FD /TC /c
+# ADD CPP /nologo /Gr /MT /W3 /GX /O1 /D "WIN32" /D "NDEBUG" /D "_WINDOWS" /YX /FD /TC /TpSource/msgcmd.cpp /c

```

Comment:
Need to make the same change to MakeFileVC for it to build on the build server:
https://github.com/diasurgical/devilution/blob/master/MakefileVC#L39

Reply:
That's the problem, it doesn't build. The change to the command line is in theory what it should be, but it doesn't work.

Reply:
In the worst case we may have to remove the `/TC` and have to specify every other file as `/Tc<file>` instead. :S